src/App.jsx

```diff
@@ -0,0 +1,44 @@
+import { Routes, Route } from 'react-router-dom';
+import { useSelector } from 'react-redux';
+import Navbar from './components/Navbar';
+import HousesList from './components/HousesList';
+import Reservation from './components/Reservation';
+import MyReservations from './components/MyReservations';
+import HouseReservations from './components/HouseReservations';
+import AddHouse from './components/AddHouse';
+import DeleteHouse from './components/DeleteHouse';
+import ShowHouse from './components/ShowHouse';
+import LoginForm from './components/LoginForm';
+import Splash from './components/Splash';
+import Logout from './components/Logout';
+import SignupForm from './components/SignupForm';
+
+function App() {
+  const { loginToken } = useSelector((store) => store.login);
+  const { logoutToken } = useSelector((store) => store.logout);
+  let isAuthorized = false;
+  if (loginToken || logoutToken) {
+    isAuthorized = true;
+  }
+  console.log('authorized', isAuthorized);
```

Comment:
According to the [Javascript best practices general requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md):

![image](https://github.com/jmonto55/book-an-appointment-frontend/assets/57421823/0c0c4279-bc25-4421-a47b-27bf80125d55)

Therefore:

- [ ] Kindly, remove the debugging code `console.log` present in this file.  

- [ ] Also, kindly remove the debugging code `console.log` present in these files:

![image](https://github.com/jmonto55/book-an-appointment-frontend/assets/57421823/9fe46bae-0b0f-4b87-8e93-ebf793635db5)

That way, your repo will be cleaner, and we will meet this particular requirement. 👍